Buildings/Controls/OBC/Utilities/PIDWithAutotuning/AutoTuner/AMIGO/Validation/PI.mo

```diff
@@ -0,0 +1,318 @@
+within Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.AMIGO.Validation;
+model PI "Test model for PI"
+  Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.AMIGO.PI PI
+    "Calculate the parameters for a PI controller"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}})));
+  Buildings.Controls.OBC.CDL.Continuous.Sources.TimeTable RefDat(table=[0,1,0.5,
```

Comment:
Putting the time series data directly in the validation model should be avoid. Please see the model `Buildings.Fluid.CHPs.Validation.ElectricalFollowing` of how to add the time series data for the validation purpose.

Reply:
I was under the impression that only the CDL components can be used in the validation. Could you please let me know if this is still the case?

Reply:
For this time series data, how did you generate them? Is it possible to implement the calculation here in the validation model (use only OBC blocks)?

Reply:
This data is generated via a Python script which is the equivalent implementation of <code>Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.AMIGO.PI</code>.

Reply:
@JayHuLBL just wanted to follow up on my previous response to see if you need more clarifications.

Reply:
I cannot see if there is anyway to log the time series efficiently. However, it seems that in the whole `AutoTuner` package the calculation is relatively simple. Like in the class `PIDGain`, it basically do the calculation `k = 1/k_p + (0.2 + 0.45T/L),`. I think there is no need to verify the Modelica calculation with Python calculation. 

I suggest to just create a simple time table with much less data point (e.g. <10 points). Or, when you generate the python data point, generate less points (10 points).

Reply:
Very good point!. I have made https://github.com/lbl-srg/modelica-buildings/pull/3326  to address this comment

Reply:
@JayHuLBL could you please let me know if more changes are needed for this issue?